test/unit/name.js

```diff
@@ -31,6 +31,7 @@ exports['Tessel.prototype.rename'] = {
   },
 
   tearDown: function(done) {
+    this.tessel.close();
```

Comment:
This fixed a memory leak where the process listener for SIGINT was never removed.